src/products/test/integ/json_request_bodies.json

```diff
@@ -6,7 +6,7 @@
     "style": "screwdriver",
     "description": "Essential screwdriver for every craftperson",
     "price": 3.99,
-    "image": "http://xxx.cloudfront.net/images/tools/8bffb5fb-624f-48a8-a99f-b8e9c64bbe29.jpg",
+    "image": "8bffb5fb-624f-48a8-a99f-b8e9c64bbe29.jpg",
```

Comment:
Image should be file name without url.  It contains URL when we call the API, which adding the URL on it.

As this content is used for update, it will corrupt the data and other tests that relies on correct image format will fail.